lib/rucio/web/rest/flaskapi/v1/common.py

```diff
@@ -425,6 +425,48 @@ def param_get(parameters: dict[str, Any], name: str, **kwargs) -> Any:
         return parameters[name]
 
 
+def param_get_bool(parameters: dict[str, Any], name: str, **kwargs) -> bool:
+    """
+        Get a boolean parameter from the passed parameters. Converts to True/False.
+    """
+    def convert_str(option: Union[str, bool]) -> bool:
```

Comment:
For clarity I would call this `convert_str_to_bool`, or maybe just `str_to_bool`. IMO I would also add an underscore at the beginning to signify that it's only used internally within the context of `param_get_bool`